bindings/pyroot_experimental/PyROOT/CMakeLists.txt

```diff
@@ -62,6 +62,9 @@ endforeach()
 ROOT_LINKER_LIBRARY(ROOTPython ${sources} LIBRARIES Core Tree cppyy)
 ROOT_INSTALL_HEADERS(inc)
 
+# Disables warnings caused by Py_RETURN_TRUE/Py_RETURN_FALSE
+target_compile_options(ROOTPython PRIVATE -Wno-strict-aliasing)
```

Comment:
This will probably break on Windows, since MSVC does not understand this option.

Reply:
Ok thanks, I will protect it, also here:
https://github.com/root-project/root/blob/master/bindings/pyroot_experimental/cppyy/CPyCppyy/CMakeLists.txt#L46